### PR TITLE
EDU-434 Update documents page table

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-434

and looks like:

Screen Shot 2022-03-28 at 10 37 10

Screen Shot 2022-03-28 at 10 37 29

### CODE REVIEW

File: src/resources/common.yml [hunk 1]
```diff
@@ -226,3 +232,15 @@ sortedBySetOn:
...
   de: Nach Typ sortiert
+sortedByUser:
+  en: Sorted by user
+  de: Nach Benutzer sortiert
+sortedByOrigin:
+  en: Sorted by origin
+  de: Nach Quelle sortiert
+sortedByArchived:
+  en: Sorted by archived status
+  de: Nach Archiviert Status sortiert
```
Comment (ahelmberger) [inline src/resources/common.yml:243]: better: "Nach Archivierungsstatus sortiert"

Comment (adrianaluchian) [reply src/resources/common.yml]: Updated, thanks!

File: src/components/document-info-cell.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+createdOn:
+  en: Created
+  de: Erstellt
+updatedOn:
+  en: Last update
+  de: Letzte Änderung
```
Comment (ahelmberger) [inline src/components/document-info-cell.yml:6]: All their brothers and sisters mostly live in `common.yml` (but maybe those are used more often than once), I wouldn't mind having those two there, too, but just a "gut feeling", no urge.

**Review by ahelmberger — APPROVED** [review]

File: src/components/document-info-cell.yml (hunk 2 above)
Comment (adrianaluchian) [reply src/components/document-info-cell.yml:6]: These two are more custom translations, they diverge from the common (they are shorter versions basically).

**Pull request merged**